src/content/en/tools/chrome-devtools/_toc.yaml

```diff
@@ -41,7 +41,7 @@ toc:
         path: /web/tools/chrome-devtools/device-mode/testing-other-browsers
     - title: Remote Debugging Android Devices
       section:
-      - title: Overview
+      - title: Get Started
```

Comment:
This doc is a Get Started. Check the DevRel doc types. I'll be updating all the DevTools docs to be more consistent with those types

cc @Meggin